MIGRATIONS.md

```diff
@@ -29,3 +29,24 @@ mysql -ss -u root -p ttm-enlace -e "SELECT Program_Date_ID, Program_ID FROM Prog
 done
 
 ```
+
+## Issue 218 - Enlace - Hiding past data - September 2018
+
+Adds a settings table for the enlace subsystem.  This is a generic placeholder for any settings that can be updated by administrators that affect how the application works.  The relevant page is system/settings.php
+
+### SQL for Table Creation
+```
+USE ttm-enlace
+CREATE TABLE `System_Settings` (
+    `Setting_Name` varchar(255) NOT NULL UNIQUE,
+    `Setting_Type` enum('integer', 'string') NOT NULL,
+    `Setting_Value` varchar(255)
+);
+```
+
+### num_days_hidden setting
+System setting for number of days, for which if a program's sessions all end after this, will be hidden.
```

Comment:
I've read that doc sentence 10 times, and I confess I don't understand what it means.  Can you say it with more words?

Reply:
Yeah, I'm updating it.  I simultaneously understand what I wrote and understand that it's not really understandable.